tests/bluetooth/mesh_shell/src/main.c

```diff
@@ -37,6 +37,10 @@ static struct bt_mesh_sol_pdu_rpl_cli srpl_cli;
 static struct bt_mesh_od_priv_proxy_cli od_priv_proxy_cli;
 #endif
 
+#if defined(CONFIG_BT_MESH_LARGE_COMP_DATA_CLI)
+struct bt_mesh_large_comp_data_cli large_comp_data_cli;
```

Comment:
I didn't get why is it required to have two instances of the large composition data client. One here and one here: https://github.com/zephyrproject-rtos/zephyr/pull/56109/files#diff-42905dbde6ea39ed1c83d8b8c3e02994a406fd25479769b71dab84ee2ef0e180R30

Reply:
Removed one.